tests/modules/nf-core/kofamscan/test.yml

```diff
@@ -0,0 +1,14 @@
+- name: "kofamscan"
+  command: nextflow run ./tests/modules/nf-core/kofamscan -entry test_kofamscan -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/kofamscan/nextflow.config
+  tags:
+    - "kofamscan"
+  files:
+    - path: "output/kofamscan/versions.yml"
+      md5sum: 4f2bd49393c4bedd7756301da6b9e826
```

Comment:
```suggestion
```

The newer versions of `nf-core` tools automatically check for the `versions.yml`. No need to check for them here too :)